dell/xps/13-9360/qca6174-firmware.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "${target}-firmware-${version}";
+  version = "${branch}-00042";
+
+  branch = "4.4.1.c1";
+  target = "QCA6174";
+
+  src = fetchurl {
+    url = "https://github.com/kvalo/ath10k-firmware/raw/master/${target}/hw3.0/${branch}/firmware-6.bin_RM.${version}-QCARMSWP-1";
```

Comment:
This hash will break if `https://github.com/kvalo/ath10k-firmware` is updated. It is better to reference a concrete commit.

Reply:
Also we can add this to nixpkgs directly.

Reply:
Fixed in master.